modules/n1ql/pages/n1ql-language-reference/numericfun.adoc

```diff
@@ -4,102 +4,127 @@
 
 {description}
 
-ABS(expression)
+== ABS(expression)
 
+=== Description
 Returns absolute value of the number.
 
-ACOS(expression)
+== ACOS(expression)
 
-Returns arccosine in radians.
+=== Description
+Returns arc cosine in radians.
 
-ASIN(expression)
+== ASIN(expression)
 
-Returns arcsine in radians.
+=== Description
+Returns arc sine in radians.
 
-ATAN(expression)
+== ATAN(expression)
 
-Returns arctangent in radians.
+=== Description
+Returns arc tangent in radians.
 
-ATAN2(expression1, expression2)
+== ATAN2(expression1, expression2)
 
-Returns arctangent of expression2/expression1.
+=== Description
+Returns arc tangent of expression2/expression1.
 
-CEIL(expression)
+== CEIL(expression)
 
+=== Description
 Returns smallest integer not less than the number.
 
-COS(expression)
+== COS(expression)
 
+=== Description
 Returns cosine.
 
-DEGREES(expression)
+== DEGREES(expression)
 
+=== Description
 Returns radians to degrees.
 
-E()
+== E()
 
+=== Description
 Base of natural logarithms.
 
-EXP(expression)
+== EXP(expression)
 
+=== Description
 Returns e^expression^.
 
-LN(expression)
+== LN(expression)
 
+=== Description
 Returns log base e.
 
-LOG(expression)
+== LOG(expression)
 
+=== Description
 Returns log base 10.
 
-FLOOR(expression)
+== FLOOR(expression)
 
-Largest integer not greater than the number.
+=== Description
+Largest integer that is not greater than the number.
 
-PI()
+== PI()
 
 Returns PI.
 
-POWER(expression1, expression2):
+== POWER(expression1, expression2)
 
+=== Description
 Returns expression1^expression2^.
 
-RADIANS(expression)
+== RADIANS(expression)
 
+=== Description
 Returns degrees to radians.
 
-RANDOM([ expression ])
+== RANDOM([ expression ])
 
+=== Description
```

Comment:
Even though it is a well-known function, it would be great if we could add usage examples per function